app/addons/documents/routes-rev-tree.js

```diff
@@ -0,0 +1,48 @@
+define([
+  "app",
+  "api",
+  "addons/documents/resources",
+  "addons/documents/views-rev-tree",
+  "addons/documents/revisionTree/actions",
+  "addons/documents/revisionTree/component.react"
+
+],
+
+function (app, FauxtonAPI, Resources, Views, RevisionTreeActions, AppComponent) {
+  var  RevTreeRouteObject = FauxtonAPI.RouteObject.extend({
+    layout: "one_pane",
+
+    crumbs: [
+      {"name": "Revision Tree" , "link": "_revtree"}
+    ],
+
+    routes: {
+       "_revtree/:database/:doc/:winner": "revtreeRoute"
```

Comment:
`:database/:doc/_revtree/:winner` is probably better, because the revtree is a part of the doc which is part of the db